docs/kb/advanced-certificates.md

```diff
@@ -63,3 +63,103 @@ LS0tLS1CRUd......==
     * Custom VMware Event Broker Appliance TLS Certificate Authority Certificate (Base64)
 
 3. Power on the VMware Event Broker Appliance and ensure that the provided TLS certificate is now used instead of the auto-generated self-sign TLS certificate by opening a browser to one of the VMware Event Broker Appliance endpoints such as `/status`.
+
+## Adding a Trusted Root Certificate to VEBA
+
+For organizations that require the use of a Private or Enterprise Certificate Authority (CA) and wish to have full TLS trust when the VMware Event Broker Appliance is configured with a vCenter Server that has a TLS certificate generated by the internal CA, a root of trust must be established.
+
+To support this use case, a post-deployment reconfiguration of the VMware Event Broker Appliance is required to import the trusted root certificate of your internal certificate authority. If this is not performed, you may see the following warning in the VMware Event Router logs when connecting to the configured vCenter Server.
+
+```console
+WARN    [VCENTER]       vcenter/vcenter.go:112  using potentially insecure connection to vCenter  {"address": "https://vcsa.primp-industries.local", "insecure": true}
+```
+
+
+In order to import your trusted root certificate, please follow the steps outlined below.
+
+### Assumptions
+
+* Pre-download your root certificates from your internal certificate, which must be .PEM encoded
+
+In the example, the downloaded root certificate file is named `ca.crt`
+
+### Steps
+
+Step 1 - Transfer or copy the contents of the root certificate to the VMware Event Broker Appliance and create a new kubernetes configMap from that file.
+
+```console
+kubectl -n vmware-system create cm ca-cert --from-file ca.crt
+```
+
+Step 2 - Undeploy the VMware Event Router and delete the current configuration secret.
+
+```console
+kubectl -n vmware-system delete -f /root/config/event-router-k8s.yaml
+kubectl -n vmware-system delete secret event-router-config
+```
+
+Step 3 - Edit `/root/config/event-router-k8s.yaml` and append the additional `volumenMount` and `volume` entry as shown in the snippet below.
+
+```console
+snip
+...
+          volumeMounts:
+            - name: config
+              mountPath: /etc/vmware-event-router/
+              readOnly: true
+            - name: ca-cert
+              mountPath: /etc/vmware-event-router/ssl
+              readOnly: true
+      volumes:
+        - name: config
+          secret:
+            secretName: event-router-config
+        - name: ca-cert
+          configMap:
+            name: ca-cert
+```
+
+Step 4 - Edit `/root/config/event-router-config.yaml` and append the additional `certificates` section to the end of the file as shown in the snippet below.
+
+```console
+snip
+...
+metricsProvider:
+  default:
+    bindAddress: 0.0.0.0:8082
+  name: veba-metrics
+  type: default
+certificates:
+  rootCAs:
+    - /etc/ssl/certs/ca-certificates.crt
+    - /etc/vmware-event-router/ssl/ca-cert.crt
```

Comment:
IMHO this is still wrong. If the filename above is `ca.crt` the projected file name would be the same, no?

Reply:
I think I fully grok it now, I may have miss-understood on the naming. I've made it a bit more clear now. Let me know if this is good